src/main/resources/application-int.yml

```diff
@@ -6,3 +6,7 @@ logging:
     com:
       jcraft:
         jsch: DEBUG
+jobs:
+  batch-print-requests:
+    daily-limit: 20
```

Comment:
I don't think I'd do this in `int` - Vijay is still integration testing with the Print Provider in `int`, and they are sending more that 20 print requests per day.
Doing it in `dev` is fine, but make sure you let the QA community know it's set that way so that a) they understand how to test this story and b) the understand why print requests might not be batched as expected if they are not testing this story! 😁

Reply:
Thanks, @NathanRussellValtech ,
I will get in touch with Vijay, to set a sensible limit for the int environment.

Reply:
@NathanRussellValtech , after talking to Vijay, we agreed on the daily limits to be 10 for dev and 1000 for int respectively. 
He said he will first look into it in dev, and 10 will be easier to test. 
I will make the changes accordingly.